pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## Status
```

Comment:
A status of the PR makes sense, but there are already mechanisms in place to handle this. If the pull request is still in development, it can be created/set to a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).

Reply:
I agree on that, and we should probably include that in the to be made contribution.md as Draft Pull Requests is quite new and not everyone know about that yet.